src/PageContent/Directory/DirectoryPage.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import DirectoryCard from './DirectoryCard.js';
+import DirectoryFilter from './DirectoryFilter.js';
+import './Directory.css';
+import "typeface-roboto";
+
+// Component for directory card
+
+class DirectoryPage extends Component {
+
+    render() {
+		return (
+            <div className="directoryPage">
+                {/* TODO: For now, I'll keep the filter component separate
+                            but I think I will move it to the Directory Page so that it will be easier
+                            to grab the filter query and use it to display the directory cards below */}
+                <DirectoryFilter />
```

Comment:
1. In the constructor, create a `receivingAgencies` and a `delivererGroups` lists in `state` that are filled with dummy data (`id` and `name` data). Just like [here](https://github.com/huangjoyce3/uw-foodwaste/blob/979872d9d066ce5f53ff6d473befa6fa21d73e6c/src/PageContent/RequestPickup/RecurringPickupRequest.js#L15-L22) in Joyce's branch.
2. Pass the list down to `DirectoryFilter` and populate the dropdown list like [here](https://github.com/huangjoyce3/uw-foodwaste/blob/979872d9d066ce5f53ff6d473befa6fa21d73e6c/src/PageContent/RequestPickup/RecurringPickupRequest.js#L222-L230).